dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/Ds3DeleteBucketTaskTest.java

```diff
@@ -0,0 +1,124 @@
+    private static final Ds3Client client = Ds3ClientBuilder.fromEnv().withHttps(false).build();
+    private static final Ds3ClientHelpers HELPERS = Ds3ClientHelpers.wrap(client);
+    private static final String TEST_ENV_NAME = "Ds3DeleteBucketTaskTest";
+    private static final String DELETE_BUCKET_TASK_TEST_BUCKET_NAME = "Ds3DeleteBucketTaskTest_Bucket";
+    private static TempStorageIds envStorageIds;
+    private static UUID envDataPolicyId;
+    private static final BuildInfoServiceImpl buildInfoService = new BuildInfoServiceImpl();
+
+    @BeforeClass
+    public static void startup() throws IOException {
+        envDataPolicyId = IntegrationHelpers.setupDataPolicy(TEST_ENV_NAME, false, ChecksumType.Type.MD5, client);
+        envStorageIds = IntegrationHelpers.setup(TEST_ENV_NAME, envDataPolicyId, client);
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException {
+        IntegrationHelpers.teardown(TEST_ENV_NAME, envStorageIds, client);
+        client.close();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        new JFXPanel();
+        Platform.runLater(() -> {
+            final SavedSession savedSession = new SavedSession(
+                    TEST_ENV_NAME,
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(
+                            client.getConnectionDetails().getCredentials().getClientId(),
+                            client.getConnectionDetails().getCredentials().getKey()),
+                    false,
+                    false);
+            session = new CreateConnectionTask().createConnection(
+                    SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+            try {
+                HELPERS.ensureBucketExists(DELETE_BUCKET_TASK_TEST_BUCKET_NAME, envDataPolicyId);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        });
+    }
+
+    @Test
+    public void deleteBucket() throws Exception {
+        final CountDownLatch latch = new CountDownLatch(1);
+        Platform.runLater(() -> {
+            try {
+                final Ds3DeleteBucketTask deleteBucketTask = new Ds3DeleteBucketTask(session.getClient(),
+                        DELETE_BUCKET_TASK_TEST_BUCKET_NAME);
+                workers.execute(deleteBucketTask);
+                deleteBucketTask.setOnSucceeded(event -> {
+                    successFlag = true;
+                    latch.countDown();
+                });
+                deleteBucketTask.setOnFailed(event -> {
+                    latch.countDown();
+                });
+                deleteBucketTask.setOnCancelled(event -> {
+                    latch.countDown();
+                });
+            } catch (final Exception e) {
+                e.printStackTrace();
```

Comment:
If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.